### PR TITLE
Make QmlStreamer compatible with Qml WebEngineView

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ Changelog {#Changelog}
 
 ### 0.12.0 (git master)
 
+* [123](https://github.com/BlueBrain/Deflect/pull/123):
+  QmlStreamer is now compatible with Qml WebEngineView items. Users must call
+  QtWebEngine::initialize() in their QApplication before creating the stream.
+  To recieve keyboard events, the objectName property of a WebEngineView must
```
Comment (dnachbaur) [inline doc/Changelog.md]: typo 'recieve'

File: deflect/qt/QmlStreamerImpl.cpp [hunk 2]
```diff
@@ -39,21 +39,29 @@
...
 #include <QQuickWindow>
 
+#if DEFLECT_USE_QT5GUI
+#include <QtGui/private/qopenglcontext_p.h>
+#endif
+
 namespace
 {
 const std::string DEFAULT_STREAM_ID( "QmlStreamer" );
+const QString GESTURES_CONTEXT_PROPERTY( "deflectgestures" );
```
Comment (dnachbaur) [inline deflect/qt/QmlStreamerImpl.cpp:63]: Mention and explain these constants in the public documentation of QmlStreamer?

File: doc/Changelog.md (hunk 1 above)
Comment (rdumusc) [inline doc/Changelog.md]: done

File: deflect/qt/QmlStreamerImpl.cpp (hunk 2 above)
Comment (rdumusc) [inline deflect/qt/QmlStreamerImpl.cpp:63]: done

**Comment by dnachbaur** [comment]
+1

**Pull request merged**